nri-kafka/src/Kafka/Worker/Internal.hs

```diff
@@ -108,7 +108,7 @@ subscription topic callback =
 subscriptionManageOwnOffsets ::
   (Aeson.FromJSON msg, Aeson.ToJSON msg) =>
   Text ->
-  ([OffsetParams] -> Task Text (Dict.Dict Int Int)) ->
+  ([OffsetParams] -> Task Text (Dict.Dict OffsetParams Int)) ->
```

Comment:
Much better!

Reply:
Alternatively, we can add the offset back onto `OffsetParams` and return a list of them instead of a dictionary.

Reply:
Did this!